tests/correlator_test.cpp

```diff
@@ -0,0 +1,485 @@
+// QUDA headers
```

Comment:
Delete this file - orthogonal to this PR

Reply:
Done in d62ea10bddc4bdf8621ec711c5adf628f13913ff